### ISSUE TITLE
Succesfully uploaded Ethereans in Kovan Test Net

### ISSUE THREAD

**Issue by alvinveroy** [MNLBlockDevs/ethereans-game#3]
https://kovan.etherscan.io/address/0x881ffaf9edac83d95c5f7c04031aae6cc0b88868

The contract is verified and tested.

**Comment by chris247474** [comment]
Looks good and pretty standard! I’ll just leave this link here for a while so others can take a look

**Comment by alvinveroy** [comment]
Yes, I made it simple so those who are still learning can easily understand it. Hope we can build more devs on this initiative. 
Thanks.
